content/components/_all.yml

```diff
@@ -1546,16 +1543,94 @@ navigation:
     - name: Patrick De Young
       img: patrick-de-young.png
       url: 'https://github.com/pattyde'
-    - name: Gary Broadbent
-      img: gary-broadbent.jpg
-      url: 'https://github.com/gazhands'
+  state: inprogress
+  order: 10
+  discussions:
+    - open: 2
+    - active: 7
+    - something: something
+  ID: main-nav    # <-- auto generated, don't change
+  version: 0.0.0    # <-- auto generated, don't change
+  dependencies:    # <-- auto generated, don't change
+    - core
+  hasSass: true    # <-- auto generated, don't change
+  hasJs: false    # <-- auto generated, don't change
+  hasReact: true    # <-- auto generated, don't change
+side-nav:
+  name: Side nav
+  description: A list of links next to the body content
```

Comment:
Could we expand on the description a bit? How is this different from a `link list`?

> A horizontal list of links for site navigation typically placed next to the body content. Can be a better choice than the `link list` component depending on context.

:point_up:  just a suggestion to build on...

Reply:
Looks good but it's vertical

Reply:
Yeah... that's what I meant...

Good pickup ;)